playbook.yaml

```diff
@@ -0,0 +1,92 @@
+- name: Install ToDo App on new web servers
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+    - name: TRELLO_API_KEY
+      prompt: What is your Trello API key?
+      private: false
+
+    - name: TRELLO_API_TOKEN
+      prompt: What is your Trello API token?
+      private: false
```

Comment:
This is totally reasonable with our current deployment structure, but it does put us at risk of the token being printed to the logs in a more sensitive environment later - e.g. if someone was told to port this into a CD pipeline, it's very plausible to me that this could end up with the token being handled carelessly, so I'd be inclined to set this to `private: true` now to program defensively